### ISSUE TITLE
multi-region: mariab-extended. Should use a single stemcell alias

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#922]
### Expected behavior

* As a paas-templates operator
* In order to 
  * keep control of stemcell used for COAB bosh deployments
  * even if a more recent stemcell is present in the bosh-coab director
* I need 
   * to define a single, iaas independent,  stemcell 'default' in the multi-region bosh manifests

### Observed behavior

2 stemcell are defined. COA 4 is only able to force the version of default stemcell, resulting in the following manifest once deployed:

```
stemcells:
- alias: default
  os: ubuntu-xenial
  version: "621.69"
- alias: openstack-hws
  name: bosh-openstack-kvm-ubuntu-xenial-go_agent
  version: latest

```

 
### Affected releases

* 48.0.1
* 47.0.3

**Comment by poblin-orange** [comment]
TODO:
- recheck hybrid stemcell bosh deploy with a single default. eg: with redis extended on brmc-int / mysql-extended on brmc-preprod.

**Comment by JCL38-ORANGE** [comment]
single alias stemcell for vsphere and openstack-hws is OK (deploy and create from scratch). It will be included in 48.0.2 tag.
